docsite/posts/config/json_yaml.md

```diff
@@ -67,6 +67,9 @@ This is the base LLM configuration section. Other steps may override this config
 - `max_retry_wait` **float** - The maximum backoff time.
 - `sleep_on_rate_limit_recommendation` **bool** - Whether to adhere to sleep recommendations (Azure).
 - `concurrent_requests` **int** The number of open requests to allow at once.
+- `temperature` **float** - The temperature to use.
+- `top_p` **float** - The top-p value to use.
+- `n` **int** - The number of completions to generate.
```

Comment:
I feel like single-letter params may be confusing on command line, since they are also used as shortcuts for longer names. Should we consider something like "completions", "n_completions", or "completions_count"?

Reply:
Sure, I can change to a more lengthy description. That said, whenever we build the dictionary it will need to be "n" as this is the parameter expected by the LLM API

Reply:
I see - in that case it is probably less confusing to align with the LLM API

Reply:
Oh btw, this isn't used on cli, just on config. And, if stored in a env var, i would be named `GRAPHRAG_LLM_N`